maths/fibonacci_sequence_recursion.py

```diff
@@ -1,10 +1,7 @@
 # Fibonacci Sequence Using Recursion
 
 def recur_fibo(n):
-    if n <= 1:
-        return n
-    else:
-        (recur_fibo(n-1) + recur_fibo(n-2))
+    return n if n <= 1 else recur_fibo(n-1) + recur_fibo(n-2)
 
 def isPositiveInteger(limit):
```

Comment:
This function is too simple to be useful.  Please removed it and do the test inline.